src/langs/constants.ts

```diff
@@ -366,4 +366,7 @@ export const BUILDER = {
   // temporary message
   "You can also find the items of this page in ''My Graasp''. This page will be unavailable soon.":
     "You can also find the items of this page in ''My Graasp''. This page will be unavailable soon.",
+  BACK: 'BACK',
```

Comment:
I think the translations for the back button are not working as expected ... please check